### PR TITLE
Data Release 2023-09-26

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: internal/server/v1/page/golden/place_page/county.Overview.json [hunk 1]
```diff
@@ -22471,12 +22471,12 @@
...
+      "value": 39029300,
       "metadata": {
-        "importName": "USCensusPEP_Annual_Population",
-        "provenanceUrl": "https://www2.census.gov/programs-surveys/popest/tables",
-        "measurementMethod": "CensusPEPSurvey",
+        "importName": "OECDRegionalDemography",
+        "provenanceUrl": "https://stats.oecd.org/Index.aspx?DataSetCode=REGION_DEMOGR#",
+        "measurementMethod": "OECDRegionalStatistics",
         "observationPeriod": "P1Y"
       }
```
Comment (pradh) [inline internal/server/v1/page/golden/place_page/county.Overview.json:22481]: We might reconsider this... whether we return OECDRegionalDemography population data for US country and states.   Might also have search implications.

Comment (pradh) [reply internal/server/v1/page/golden/place_page/county.Overview.json:22481]: Or maybe we need to update USCensusPEP?

Comment (pradh) [reply internal/server/v1/page/golden/place_page/county.Overview.json:22481]: It is already at the top - https://github.com/datacommonsorg/mixer/blob/d7022933628a6df248842981d3cc75cbad697b8e/internal/server/ranking/ranking.go#L45-L48.  Any ideas why we OECD shows up for US places?

Comment (shifucun) [reply internal/server/v1/page/golden/place_page/county.Overview.json:22481]: When choosing the population series in place page, the facet with latest data is used indeed: https://github.com/datacommonsorg/mixer/blob/d7022933628a6df248842981d3cc75cbad697b8e/internal/server/v1/page/place.go#L331
Shall we not doing this?

Comment (shifucun) [reply internal/server/v1/page/golden/place_page/county.Overview.json:22481]: Sent a PR to test this out: https://github.com/datacommonsorg/mixer/pull/1275/files

Comment (shifucun) [reply internal/server/v1/page/golden/place_page/county.Overview.json:22481]: The PR is merged. @chejennifer can you sync and refresh the golden files?

Comment (chejennifer) [reply internal/server/v1/page/golden/place_page/county.Overview.json]: updated, ptal

**Review by pradh — APPROVED** [review]
Looks good to me!  Thanks for the fixes Bo and Jennifer!

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**